src/sentry/utils/sentry_apps/webhooks.py

```diff
@@ -49,7 +50,12 @@ def check_broken(sentryapp: SentryApp, org_id: str):
             sentryapp._disable()
             notify_disable(org, sentryapp.name, redis_key, sentryapp.slug, sentryapp.webhook_url)
             buffer.clear()
-
+            create_system_audit_entry(
```

Comment:
I saw this in the audit utils and gave it a shot - you can add this to the other disable spot too.